exchange/events.go

```diff
@@ -0,0 +1,89 @@
+package exchange
+
+import (
+	"encoding/json"
+	"time"
+
+	jsonpatch "github.com/evanphx/json-patch"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/analytics"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/endpoints/events"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/pbsmetrics"
+)
+
+// eventsData has configuration fields needed for adding event tracking to an auction response
+type eventsData struct {
```

Comment:
Nitpick: Consider renaming to `eventTracking`?

Reply:
Will do. I had named it `eventsData` to parallel `targetingData` which has similar interface.